scripts/linters/pre_commit_linter.py

```diff
@@ -557,6 +558,9 @@ def main(args=None):
         third_party_typings_linter.check_third_party_libs_type_defs(
             verbose_mode_enabled))
 
+    lint_messages += webpack_config_linter.check_webpack_config_file(
+        FILE_CACHE, verbose_mode_enabled)
```

Comment:
Instead of passing the whole file_cache can we just pass the specific file? (not sure!)

Also, considering this file is only checked by this test can we remove this param and webpack_config_linter can directly fetch it like the way its done in other two test above?

Reply:
codeowner_linter also use FILE_CACHE, this is fast in multiprocessing instead of opening file directly.